assets/src/styles/clonerbookcard.scss

```diff
@@ -0,0 +1,105 @@
+.ais-Hits-item {
```

Comment:
It was a compiled version of the current pressbooks network catalog style. I am sure better techniques and styles must be applied. Also, it does not meet the defined UI specs.